src/m365/spo/commands/navigation/navigation-node-list.ts

```diff
@@ -24,6 +24,10 @@ class SpoNavigationNodeListCommand extends SpoCommand {
     return 'Lists nodes from the specified site navigation';
   }
 
+  public defaultProperties(): string[] | undefined {
+    return ['AudienceIds', 'CurrentLCID', 'Id', 'IsDocLib', 'IsExternal', 'IsVisible', 'ListTemplateType', 'Title', 'Url'];
```

Comment:
I think my last review was not clear enough, let me try again. 😃 
I think that we should keep the old output (Id, Title and Url) as deafultProperties output. JSON and MD can contain all properties, for Text and CSV I'd keep the current result.